content/_index.html

```diff
@@ -314,6 +319,10 @@ <h1 style="color: white;">
               src="/img/palette.svg"
               alt="Style palette"
             />
+
+            <div
+              style="background-image: url('/img/maputnik.png'); height: 300px; width:100%;max-width: 150px; background-size:contain; background-position: center; background-repeat: no-repeat;"
```

Comment:
should this be a link too?

Reply:
it's a link now. none of the other images are links yet.